src/devices/bus/segaai/segaai_exp.h

```diff
@@ -0,0 +1,61 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+#ifndef MAME_BUS_SEGAAI_EXP_H
+#define MAME_BUS_SEGAAI_EXP_H
+
+#pragma once
+
+
+DECLARE_DEVICE_TYPE(SEGAAI_EXP_SLOT, segaai_exp_slot_device);
+
+
+class segaai_exp_slot_device : public device_t,
+								public device_slot_interface
```

Comment:
Any reason not to use `device_single_card_slot_interface` and pick up the validation helpers automatically?  You generally only want to derive from `device_slot_interface` directly if you have cards with multiple interfaces.

Reply:
Changed to use device_single_card_slot_interface.